scripts/gui/microbe_editor.mjs

```diff
@@ -1172,66 +1189,105 @@ function processPatchMapData(data){
 function updateSelectedPatchData(patch){
     selectedPatch = patch;
 
+    // Reset species shown
+    document.getElementById("speciesInPatch").textContent = "";
+
     if(!selectedPatch){
         document.getElementById("noPatchSelectedText").style.display = "inline-block";
         document.getElementById("patchInfoBox").style.display = "none";
         document.getElementById("moveToPatchButton").classList.add("Disabled");
         return;
     }
 
+    // Get chunck values of patch
+    const chunk = getPatchChunk(patch);
+
     document.getElementById("noPatchSelectedText").style.display = "none";
     document.getElementById("patchInfoBox").style.display = "block";
 
     document.getElementById("editorSelectedPatchName").textContent = patch.name;
 
-    const descriptionElement =
-        document.getElementById("editorSelectedPatchDescription");
-    descriptionElement.textContent = "";
-
     if(patchMoveAllowed(selectedPatch.id)){
         document.getElementById("moveToPatchButton").classList.remove("Disabled");
     } else {
         document.getElementById("moveToPatchButton").classList.add("Disabled");
     }
 
+    // Reset all arrows and text when we select a patch
+    // Is called when we move on another patch or we select one
     if(currentPatchId == selectedPatch.id){
-        descriptionElement.appendChild(document.createTextNode("You are currently in " +
-                                                               "this patch."));
-        descriptionElement.appendChild(document.createElement("br"));
+        document.getElementById("editorSelectedPatchName").textContent =
```

Comment:
I think that this "you are here" text should have its own element. I don't like that you can't see the name of the patch you are in.